extensions/aws2-s3/deployment/src/main/java/org/apache/camel/quarkus/component/aws2/s3/deployment/Aws2S3Processor.java

```diff
@@ -89,12 +89,6 @@ void client(BeanRegistrationPhaseBuildItem beanRegistrationPhase,
             BuildProducer<NativeImageProxyDefinitionBuildItem> proxyDefinition) {
         checkClasspath(APACHE_HTTP_SERVICE, "apache-client");
```

Comment:
Isn't this class path check also a good candidate to move to the new support extension?

Reply:
Yes, but actually it doesn't seem to work when moved to commons. So lets have a skeleton for the moment and adding more extensions and then revisit the support extension, if it's ok for all, obviously